java/fury-core/src/main/java/org/apache/fury/resolver/BlackList.java

```diff
@@ -50,7 +51,9 @@ class BlackList {
     }
   }
 
-  static Set<String> getDefaultBlackList() {
-    return DEFAULT_BLACKLIST_SET;
+  static void checkNotInBlackList(String clsName) {
```

Comment:
Could we document this method that it will throws exception when class is in blacklist

Reply:
Updated.